app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```

Comment:
Can these edits be added profile page instead?

Reply:
Sure, but that can be a new ticket and separate issue.